tests/fast-integration/skr-svcat-migration-test/skr-svcat-migration-test.js

```diff
@@ -0,0 +1,69 @@
+const uuid = require("uuid");
+const { 
+  KEBConfig,
+  KEBClient,
+  provisionSKR,
+  deprovisionSKR,
+} = require("../kyma-environment-broker");
+const {
+  GardenerConfig,
+  GardenerClient
+} = require("../gardener");
+const {
+  debug,
+  genRandom,
+  getEnvOrThrow,
+  initializeK8sClient,
+} = require("../utils");
+const t = require("./test-helpers");
+const fs = require('fs');
+const os = require('os');
+
+describe("SKR SVCAT migration test", function() {
+  const keb = new KEBClient(KEBConfig.fromEnv());
+  const gardener = new GardenerClient(GardenerConfig.fromEnv());
+
+  const suffix = genRandom(4);
+  const appName = `app-${suffix}`;
+  const runtimeName = `kyma-${suffix}`;
+  const runtimeID = uuid.v4();
+  
+  const svcatPlatform = `svcat-${suffix}`
+  const btpOperatorInstance = `btp-operator-${suffix}`
+  const btpOperatorBinding = `btp-operator-binding-${suffix}`
+
+  debug(`RuntimeID ${runtimeID}`, `Runtime ${runtimeName}`, `Application ${appName}`, `Suffix ${suffix}`);
+
+  const testNS = "skr-test";
+  const AWS_PLAN_ID = "361c511f-f939-4621-b228-d0fb79a1fe15";
+
+  const clientID = getEnvOrThrow("BTP_OPERATOR_CLIENTID");
+  const clientSecret = getEnvOrThrow("BTP_OPERATOR_CLIENTSECRET");
+  const URL = getEnvOrThrow("BTP_OPERATOR_URL");
```

Comment:
I think it is worth to encapsulate those variables into some sort of structure, pleas see `KEBConfig` or `GardenerConfig` as examples.